OpenRA.Game/Server/Server.cs

```diff
@@ -20,106 +20,65 @@
 using OpenRA.Primitives;
 using OpenRA.Support;
 
-using XTimer = System.Timers.Timer;
-
 namespace OpenRA.Server
 {
-	public enum ServerState : int
+	public enum ServerState
 	{
-		WaitingPlayers = 1,
-		GameStarted = 2,
-		ShuttingDown = 3
+		WaitingPlayers,
+		GameStarted,
+		ShuttingDown
```

Comment:
I'd still like to have the old default values back. You can remove the ` : int` though.